h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -674,63 +674,17 @@ public Vec cv_AssignFold(int N) {
 
   // Step 4: Run all the CV models and launch the main model
   public void cv_buildModels(int N, ModelBuilder<M, P, O>[] cvModelBuilders ) {
-    bulkBuildModels("cross-validation", _job, cvModelBuilders, nModelsInParallel(N), 0 /*no job updates*/);
+    makeBulkModelBuilder("cross-validation", _job, cvModelBuilders, nModelsInParallel(N), 0 /*no job updates*/).bulkBuildModels();
     cv_computeAndSetOptimalParameters(cvModelBuilders);
   }
-
-  /**
-   * Runs given model builders in bulk.
-   *
-   * @param modelType text description of group of models being built (for logging purposes)
-   * @param job parent job (processing will be stopped if stop of a parent job was requested)
-   * @param modelBuilders list of model builders to run in bulk
-   * @param parallelization level of parallelization (how many models can be built at the same time)
-   * @param updateInc update increment (0 = disable updates)
-   */
-  public static void bulkBuildModels(String modelType, Job job, ModelBuilder<?, ?, ?>[] modelBuilders,
```

Comment:
This method was originally built for CV models only and then made public for MLI. I don't think it is used in MLI anymore.

Reply:
@dvorka please confirm the above or review if this change would break MLI

Reply:
@navdeep-G ^

Reply:
This is not used in MLI